sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/LdapConnectionShouldBeSecure.cs

```diff
@@ -129,4 +129,64 @@ void Cases()
             Property6 = new DirectoryEntry(); // Compliant
         }
     }
+
+    public class AllTypes
```

Comment:
can you also add test cases (unless I miss them) for initializations inside
- constructor
- inside getter / setter (for properties we only have `private DirectoryEntry Property3 { get; set; } = new DirectoryEntry()...`)
- struct
- interface default method